package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Production server for running Ember applications using FastBoot",
   "main": "./lib/server",
   "scripts": {
+    "postinstall": "scripts/is-not-legacy-vm || npm install contextify@^0.1.11",
```

Comment:
This isn't working as I would expect.  `is-not-legacy-vm` is reporting as new in every version of node I've tried (back to 0.12).  So it never falls through to the conditional install.  `contextify` is installed, but that's because it's still in the package.json.